server/rest/postgres_geojson.py

```diff
@@ -93,6 +93,7 @@ def getValues(self, assetstore, params):
         return self._getValues(assetstore, params)
 
     def _getValues(self, assetstore, params):
+        filter = params['filter'] if 'filter' in params else None
```

Comment:
cannot you just do this filter = params.get('filter')?

Reply:
Yes, thanks